src/sentry/issues/json_schemas.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any, Mapping
+
+EVENT_PAYLOAD_SCHEMA: Mapping[str, Any] = {
+    "type": "object",
+    "properties": {
+        # required properties
+        "event_id": {"type": "string", "minLength": 1},
+        "platform": {"type": "string", "minLength": 1},
+        "project_id": {"type": "integer"},
+        "tags": {"type": "object"},
+        "timestamp": {"type": "string", "format": "date-time"},
+        "message_timestamp": {"type": "string", "format": "date-time"},
+        # "title": {"type": "string", "minLength": 1}, leaving this out, for now
+        # non-required properties
+        "breadcrumbs": {"type": "object"},
+        "contexts": {"type": "object"},
```

Comment:
This will be a `dict[str, dict[str, str]]`, not sure if we can be more explicit in this definition

Reply:
let me see what I can do